README.md

```diff
@@ -9,7 +9,8 @@
 
 Immutable collection interfaces and implementation prototypes for Kotlin.
 
-This is a multiplatform library providing implementations for `jvm`, `js` (both Legacy and IR), `mingwX64`, `linuxX64` and Apple Kotlin/Native targets.
+This is a multiplatform library providing implementations for `jvm`, `js` (both Legacy and IR),
```

Comment:
Please check the build. JS legacy is not supported in Kotlin 1.9

Reply:
@qurbonzoda could you update this?

Reply:
The JS Legacy support is now dropped